ChurchCRM/CRM
# Check cURL against self for mod_rewrite

#### What's this PR do?
Checks for the presence of a header supplied by index.php.
index.php should really only be invoked if .htaccess / mod_rewrite has directed the request. 
In the event of an invalid URL, index will be invoked.  

This makes AppIntegrityService make a request for a known non-existent URL on the current installation to verify that the header is present that we expect from index.php.

#### What Issues does it Close?

Closes #1871

#### How should this be manually tested?
1)  On a host that incorrectly states mod_rewrite is not present from master build, this pr should preport it is present
2)  Edit the .htaccess to disable ```RewriteEngine``` - the SystemPrerequisiteCheck should (correctly) report a failure.
